src/base.c

```diff
@@ -98,7 +330,14 @@ static const
 R_CallMethodDef callMethods[] = {
   {"kx_r_open_connection", (DL_FUNC) &kx_r_open_connection, -1},
   {"kx_r_close_connection", (DL_FUNC) &kx_r_close_connection, -1},
-  {"kx_r_execute", (DL_FUNC) &kx_r_execute, -1},
+  {"kx_r_execute0", (DL_FUNC) &kx_r_execute0, -1},
```

Comment:
it is automatic formatting for source code. No worries, I can do it later if you don't have it installed.